pxd.c

```diff
@@ -489,6 +489,7 @@ static void pxd_rq_fn(struct request_queue *q)
 			continue;
 		}
 
+		req->fastpath = pxd_dev->fastpath;
```

Comment:
The above check should be IS_ERR_OR_NULL too... and all callers of pxd_fuse_req should ensure its checked that way.